app/javascript/psulib_blacklight/javascripts/availability.js

```diff
@@ -10,6 +10,7 @@ var allLocations = locations.locations;
 var allLibraries = locations.libraries;
 var illiadLocations = locations.request_via_ill;
 var allItemTypes = item_types.item_types;
+const scanURL = 'https://psu.illiad.oclc.org/illiad/upm/lending/lendinglogon.html';
```

Comment:
@cdmo `scanURL` seems to be not used, maybe intended to be used in line 410: https://github.com/psu-libraries/psulib_blacklight/blob/9bfe687facb564fe268bcb22b38cad5918ee05b7/app/javascript/psulib_blacklight/javascripts/availability.js#L410

Reply:
Yes. My mistake. Sorry! Gonna close this PR, it's a mess.